helm/flink-kubernetes-operator/crds/flinkdeployments.flink.apache.org-v1.yml

```diff
@@ -9243,6 +9242,19 @@ spec:
                 type: object
               error:
                 type: string
+              lifecycleState:
+                enum:
+                - CREATED
+                - SUSPENDED
+                - UPGRADING
+                - DEPLOYED
+                - STABLE
+                - ROLLING_BACK
+                - ROLLED_BACK
+                - FAILED
+                - terminal
```

Comment:
What is this exactly? :)

Reply:
For some reason the fabric8 crd gen includes all the fields from the `ResourceLifecycleState`, and this is why `terminal` and `description` also appear. This is not good I agree, maybe some annotation needed above those fields to ignore those by the crd generator. I can take a look at those tomorrow afternoon (CET). However, maybe that is not required to commit the automated generated yaml files, am I right?